ObjectiveZipLib/Objective-Zip/UnzipWithProgress.mm

```diff
@@ -324,11 +347,18 @@ - (BOOL) extractStream:(ZipReadStream *) readStream
 {
    BOOL result = NO;
    
+   
    NSURL * fullURL = [unzipToFolder URLByAppendingPathComponent:info.name];
    
    if ([_zipDelegate respondsToSelector:@selector(updateCurrentFile:)])
       [_zipDelegate  updateCurrentFile:fullURL];
    
+   // let the delegate modify the file name.  We do this in all cases, because the delegate may have its own ideas aboutwhat a collision is
+   if ( self.unzipFileDelegate )
```

Comment:
couldn't the caller figure out the collisions before calling? like there's no new info that the caller has when this delegate method is called right?

Reply:
the delegate method is called for each file in the zip file.

Reply:
talked to s.wagner and now i get it that caller doesn't know the names of the files, that's the new info they have :)